ci-operator/config/openshift/assisted-service/openshift-assisted-service-ocm-2.3.yaml

```diff
@@ -1,19 +1,19 @@
 base_images:
   assisted-installer:
     name: ocm-2.3
-    namespace: openshift
+    namespace: edge-infrastructure
     tag: assisted-installer
   assisted-installer-agent:
     name: ocm-2.3
-    namespace: openshift
+    namespace: edge-infrastructure
     tag: assisted-installer-agent
   assisted-installer-controller:
     name: ocm-2.3
-    namespace: openshift
+    namespace: edge-infrastructure
     tag: assisted-installer-controller
   assisted-service-scripts:
-    name: assisted
-    namespace: ocp-kni
+    name: ocm-2.4
```

Comment:
Shouldn't it be 2.3?

Reply:
We should probably sync on slack about it... but we don't currently release this container for 2.3 because it came about in 2.4 cycle. I was debating between 2.4 and 2.5 but decided 2.4 is probably closer than 2.5 to 2.3 so went with that.

Reply:
OK, no biggie